activerecord/test/cases/validations/numericality_validation_test.rb

```diff
@@ -6,6 +6,9 @@
 class NumericalityValidationTest < ActiveRecord::TestCase
   def setup
     @model_class = NumericData.dup
```

Comment:
One day I'll try rewriting the test file to avoid using `.dup` as this is not the first time the test fails only because the class is duped which is not something Active Record expects